responsive/templatetags/responsive_tags.py

```diff
@@ -0,0 +1,19 @@
+from django import template
+
+register = template.Library()
+
+@register.assignment_tag(takes_context=True)
+def device_size(context):
+    """Create a variable in context containing the devices' size
+    as recorded by django-responsive2
+    
+    Usage in template: {% device_size as device_size %}
+    
+    <div class="{{ device_size }}">...</div>
+    """
+    device = context.get('device', '')
```

Comment:
Should use `settings.RESPONSIVE_VARIABLE_NAME`.
i.e `device = getattr(context, settings.RESPONSIVE_VARIABLE_NAME, None)`